### ISSUE TITLE
Narrator announces the wrong message when changing font properties in Font dialog

### ISSUE THREAD

**Issue by Philip-Wang01** [dotnet/winforms#5314]
* .NET Core Version: 
.NET 6.0.100-rc.1.21371.12

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
Narrator announces the wrong message when changing font properties in Font dialog. which is not regression issue, can reproduce on .Net Core 3.1 and .Net 5.0

**Expected behavior:**
Narrator should announce the correct information when changing font properties in Font dialog, please see following screenshot in .Net framework 4.8:
![narofont](https://user-images.githubusercontent.com/86937911/126580171-cfa6c253-2d30-4223-8be1-c926ee64e9d2.png)
![nazhongyao](https://user-images.githubusercontent.com/86937911/126580227-ce994dc5-947f-475a-baa2-1cf8b8652e38.gif)

**Actual behavior:**
Narrator announced the wrong message when changing font properties in Font dialog, please see following screenshot for .Net 6.0:
![narrooo123](https://user-images.githubusercontent.com/86937911/126580342-15b3ea75-0170-48cc-bad5-52e1374f64bb.png)
![na123465](https://user-images.githubusercontent.com/86937911/126580349-a259845e-a18f-4d4d-a93a-1f3ecf009871.gif)

**Minimal repro:**
1. Create a project that uses the FontDialog control
2. Build and run it
3. Change font properties in Font dialog
4. Open the Narrator tool, and observe its behavior.

**Comment by merriemcgaw** [comment]
@Philip-Wang01 do you know if this was a regression from an earlier version of .NET?
@Tanya-Solyanik do we create the FontDialog ourselves or do we get it from Windows?

**Comment by ArtemTatarinov** [comment]
I have reproduced this issue also in .Net Framework 4.7.2 and in MFC application too. Also, I have looked through the `FontDialog.cs` source code file and it doesn't contain any of the controls that we can see at it - no `Strikeout` and `Underline` `CheckBoxes`, no `ListBoxes` with fonts, font styles, font sizes, no `TextBoxes` with selected font, style, size, and so on. So my guess is this control is created by some inner WinAPI function and the whole issue is rooted somewhere deep inside Windows/WinAPI. @merriemcgaw, could you please tell what do think about it? 

**.Net Framework 4.7.2**
![image](https://user-images.githubusercontent.com/87859299/133777721-11ae17b9-8be7-4834-afd4-6e128c70fc1e.png)

**MFC**
![image](https://user-images.githubusercontent.com/87859299/133777799-4e4e30f7-6e10-46cf-9618-39cd9c7891b1.png)

**Comment by merriemcgaw** [comment]
It looks to me like this is a Windows problem then and we should go ahead and file a bug on them. @Tanya-Solyanik, would you agree?

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw, @ArtemTatarinov, @vladimir-krestov   - yes, this bug should be copy/pasted into the OS bug data base, on common dialog area path with images from this investigation and the MFC sample attached.

@Philip-Wang01  - we don't own content of this dialog, winforms only invokes windows (comdlg32) APIs, so we don't have to test it.

**Comment by Tanya-Solyanik** [comment]
WIndows [bug](https://microsoft.visualstudio.com/OS/_workitems/edit/39833818) is here, @merriemcgaw - should we close this issue?

**Comment by merriemcgaw** [comment]
@Tanya-Solyanik thanks!!
